human-spaceflight/data/MORGAN_AR.md

```diff
@@ -0,0 +1,35 @@
+---
+category:	astronaut
+name:	MORGAN, Andrew Richard
+type:	military
+dob:	1976-02-05
+birthplace:
+  - name:	Morgantown, West Virginia
+    latitude:	39.633611
+    longitude:	-79.950556
+qualifications:
+  - BSc in Environmental Engineering
+  - Medical Doctorate
+country:
+  - USA
+missions:
```

Comment:
**Should be:** http://www.spacefacts.de/iss/english/exp_60.htm
missions:
  - name: Soyuz MS-13
    time_start: 2019-07-20T16:28Z
    time_end:
  - name: Soyuz MS-15
    time_start:
    time_end: 2020-04-17T05:16Z

Reply:
oops - copy past error! thanks!